local-test.js

```diff
@@ -35,5 +34,7 @@ if (!firestoreData) {
     throw new Error('Problem with permission of read and write.\nCheck your firestore permissions')
   }
   await docRef.delete()
-  logger.info('Success! The backend was successfully set up.')
+
+  // eslint-disable-next-line no-console
+  console.log('Success! The backend was successfully set up.')
```

Comment:
Using console log here as using logger doesn't seem to make sense here as this is a utility task, not an application file.
It also solves the issue of missing config import in logger.